backend/apps/owasp/models/board_candidate_claim_evidence.py

```
@@ -122,12 +123,64 @@ def clean(self) -> None:
+                )
+
+                output = BytesIO()
+                writer.write(output)
+                output.seek(0)
+
+                from django.core.files.base import ContentFile
+
+                self.file = ContentFile(output.read(), name=self.file.name)
+        except Exception:
+            # TODO:  not sure i should raise error here ? or let it pass
+            pass
```

Ruff (0.15.18)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)